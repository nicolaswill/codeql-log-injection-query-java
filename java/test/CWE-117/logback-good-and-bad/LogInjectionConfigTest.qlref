CWE-117/LogInjectionConfig.ql