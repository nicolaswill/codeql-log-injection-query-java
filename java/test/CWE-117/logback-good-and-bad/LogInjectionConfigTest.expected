| logback.xml:8:5:11:12 | root | Logger $@ uses appender $@, which is vulnerable to log injection because the appender uses the pattern layout encoder. | logback.xml:8:5:11:12 | root | root | logback.xml:4:5:4:84 | appender | STDOUT |
